ResearchKit/Common/ORKOrderedTask.m

```diff
@@ -785,4 +787,58 @@ + (ORKOrderedTask *)spatialSpanMemoryTaskWithIdentifier:(NSString *)identifier
     return task;
 }
 
++ (ORKOrderedTask *)deviceMotionReactionTimeTaskWithIdentifier:(NSString *)identifier
+                            intendedUseDescription:(nullable NSString *)intendedUseDescription
+                                  getReadyInterval:(NSTimeInterval)getReady
+                            maximumStimulusInterval:(NSTimeInterval)maximumStimulusInterval
+                             minimumStimulusInterval:(NSTimeInterval)minimumStimulusInterval
+                                     terminationGs:(double)terminationGs
+                                   numberOfAttempts:(int)numberOfAttempts
+                                           timeout:(NSTimeInterval)timeout
+                                           options:(ORKPredefinedTaskOption)options {
+    
+    NSMutableArray *steps = [NSMutableArray array];
+    
+    if (! (options & ORKPredefinedTaskOptionExcludeInstructions)) {
+            ORKInstructionStep *step = [[ORKInstructionStep alloc] initWithIdentifier:ORKInstruction0StepIdentifier];
+            step.title = ORKLocalizedString(@"REACTION_TIME_TASK_TITLE", nil);
+            step.text = intendedUseDescription;
+            step.detailText = ORKLocalizedString(@"REACTION_TIME_TASK_INTENDED_USE", nil);
+            step.image = [UIImage imageNamed:@"phonewaves" inBundle:[NSBundle bundleForClass:[self class]] compatibleWithTraitCollection:nil];
+            step.shouldTintImages = YES;
+            
+            ORKStepArrayAddStep(steps, step);
+        }
```

Comment:
This doesn't look right. If excluding instructions, you want to exclude both instructions steps, not just this one. Check bracket positioning versus the examples you copied from.